IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -763,7 +763,7 @@ - (NSArray *)itemsWithKey:(MSIDCacheKey *)key
     NSData *generic = key.generic;
     NSNumber *type = key.type;
     
-    MSID_LOG_WITH_CTX_PII(MSIDLogLevelVerbose, context, @"Get keychain items, key info (account: %@ service: %@ generic: %@ type: %@, keychainGroup: %@)", MSID_EUII_ONLY_LOG_MASKABLE(account), service, generic, type, [self keychainGroupLoggingName]);
+    MSID_LOG_WITH_CTX_PII(MSIDLogLevelVerbose, context, @"Get keychain items, key info (account: %@ service: %@ generic: %@ type: %@, keychainGroup: %@)", MSID_PII_LOG_MASKABLE(account), service, generic, type, [self keychainGroupLoggingName]);
```

Comment:
Can you double check why this PR is reverting the EUII logging change?

Reply:
git merge dev didn't work. somehow ended up reverting all the log changes when I tried to merge with dev and then with Hieu's branch. something went wrong. I had to update this branch to have all of your logging changes manually.